packages/yourloops/pages/notifications/notifications-list.tsx

```diff
@@ -71,24 +71,25 @@ const NotificationsPage: FunctionComponent<PropsWithChildren> = () => {
   const { user } = useAuth()
   const notificationsHook = useNotification()
   const [switchRoleOpen, setSwitchRoleOpen] = useState<boolean>(false)
+  const notifications = notificationsHook.receivedInvitations
 
   useEffect(() => {
     setPageTitle(t('breadcrumb-notifications'))
   }, [t])
-
   if (user === null) {
     throw new Error('Notification require a logged-in user')
   }
 
-  const notifications = notificationsHook.receivedInvitations
-
   const handleSwitchRoleOpen = (): void => {
     metrics.send('switch_account', 'display_switch_notification')
     setSwitchRoleOpen(true)
   }
   const handleSwitchRoleCancel = (): void => {
     setSwitchRoleOpen(false)
   }
+  const refreshNotificationHook = async (): Promise<void> => {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏
```suggestion
  const refreshReceivedInvitations = async (): Promise<void> => {
```